app/scripts/shell/dimSearchFilter.directive.js

```diff
@@ -2,20 +2,103 @@
   'use strict';
 
   angular.module('dimApp')
-    .factory('dimSearchService', function() {
+    .factory('dimSearchService', function(dimSettingsService) {
```

Comment:
No, I was going to keep that

On Thursday, September 8, 2016, kyle shay <notifications@github.com> wrote:

> In app/scripts/shell/dimSearchFilter.directive.js
> <https://github.com/DestinyItemManager/DIM/pull/471#discussion_r78012983>:
>
> > @@ -2,20 +2,103 @@
> >    'use strict';
> >
> >    angular.module('dimApp')
> > -    .factory('dimSearchService', function() {
> > +    .factory('dimSearchService', function(dimSettingsService) {
>
> @bhollis <https://github.com/bhollis> was the intention here to rid of
> this service? i can't remember what it was before now.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/DestinyItemManager/DIM/pull/471/files/7a00da36c12276f1a7b9a5b89baf738bd03020ef#r78012983>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AATHeHJutmnVsL7cnXoBJ40t3OCDKP5yks5qoBcigaJpZM4ICrP0>
> .
>